packages/versioned-playground/src/hooks.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from "react";
+
+import { useToast } from "@quri/ui";
+
+import {
+  SquiggleVersion,
+  checkSquiggleVersion,
+  defaultSquiggleVersion,
+} from "./versions.js";
+
+export function useValidSquiggleVersion(version: string): SquiggleVersion {
```

Comment:
I might change the name to something a bit more precise, like, ``adjustToEnsureValidSquiggleVersion``

Reply:
That's quite long, especially because it needs `use` prefix since it's a hook, but I renamed it to `useAdjustSquiggleVersion`.